pkg/event/event.go

```diff
@@ -420,10 +555,10 @@ func (e *EventHandler) updateCLockClass(cfgName string, ptpState PTPState, clock
 		return err, clockClass
 	}
 	glog.Infof("current GRANDMASTER_SETTINGS_NP:\n%s", g.String())
-	switch ptpState {
-	case PTP_LOCKED:
-		switch clockType {
-		case GM:
+	switch clockType {
+	case GM:
```

Comment:
Can you add comments here what each clock class is?

Reply:
added